server/apps/immich/src/api-v1/asset/asset.controller.ts

```diff
@@ -260,6 +260,18 @@ export class AssetController {
     return await this.assetService.getAssetByTimeBucket(authUser, getAssetByTimeBucketDto);
   }
 
+  /**
+   * Get all assets that have GPS information embedded
+   */
+  @Authenticated()
+  @Get('/mapMarker')
```

Comment:
I think this should be `map-marker`. It was my lack of understanding of writing REST API endpoint name to use camelCase :)